map2loop/thickness_calculator.py

```diff
@@ -199,6 +211,7 @@ def __init__(self):
         """
         self.thickness_calculator_label = "InterpolatedStructure"
         self.lines = []
+        self.max_line_length = None
```

Comment:
Where is this initialised?

Reply:
assuming this has to do with your comment below, so will resolve this thread for now

> Why are the attributes hard coded? line length should be set per map. Make these variables of the init function with these values as default